src/components/InvestInTranche.tsx

```diff
@@ -102,6 +113,16 @@ export const InvestInTranche: FunctionComponent<InvestInTrancheProps> = ({ tranc
 		}
 	};
 
+	const validateTranchePass = (): Promise<void> => {
```

Comment:
if there is a tranche pass, check if it's active. If it's not, return a different error message